Tests/SpeziFoundationTests/OrderedArrayTests.swift

```diff
@@ -97,7 +105,7 @@ struct OrderedArrayTests {
         #expect(array.capacity == capacity)
     }
     
-    @Test
+    @Test(.disabled(if: isReleaseBuild))
```

Comment:
What's the reason that release builds won't work here?

Reply:
If it's already implemented why not add it and merge it for backwards compatibility? Or would it require a lot of additional effort?

Reply:
Maybe let's add it to make the package "feature complete" and useful for the community in the long run. But yes, as we move things to Xcode 26 and things become stable there we can slowly phase it out across packages. Will probably take us a decent amount of time anyways to having something in the transition is not bad.